kit/src/main/java/com/oracle/javafx/scenebuilder/kit/library/user/LibraryFolderWatcher.java

```diff
@@ -411,9 +411,19 @@ else if (Files.isDirectory(currentJarOrFolder)) {
                 final String canonicalName = e.getKlass().getCanonicalName();
                 if (!excludedItems.contains(canonicalName) && 
                     !artifactsFilter.contains(canonicalName)) {
-                    final String name = e.getKlass().getSimpleName();
-                    final String fxmlText = JarExplorer.makeFxmlText(e.getKlass());
-                    result.add(new LibraryItem(name, UserLibrary.TAG_USER_DEFINED, fxmlText, iconURL, library));
+                	
+                	final String name = e.getKlass().getSimpleName();
+                	String sectionName =jarOrFolderReport.getJar().toString();
+                	
+                	// if some os don't use '/' for folder path (don't know if there is)
+                	if (sectionName.lastIndexOf("\\")==-1)
+                		sectionName=UserLibrary.TAG_USER_DEFINED;
+                	else
+                		sectionName=sectionName.substring(sectionName.lastIndexOf("\\")+1, 
+                					sectionName.lastIndexOf("."));
```

Comment:
what if there is no `.` in the name, or it occurs before the last `'` . This should not happen, but in that case, it makes sense to fallback to the default?